### DISCUSSION TITLE
Extensions

### DISCUSSION THREAD

**Discussion by Takemichi00** [aniyomiorg/aniyomi-extensions#47] [Ideas]
Aniyomi needs more anime extensions , with 4anime gone there are only 2 anime extensions that work and 1hentai extension , if it is possible , I suggested creating more extensions:
9anime.to extension ;
Zoro.to .

**Comment by impulsado** [comment]
I think the same, I think there are very few extensions. I would ask if you can put some in Spanish like "animeflv.net".

Even so very good job, I come from using Tachiyomi and this seems to me impressive.🧨🧨

**Comment by DeCode39** [comment]
Is it possible to add an extension for animepahe?

**Reply by jmir1** [reply]
Yes it's done now

**Reply by DeCode39** [reply]
App crashes while trying to play.

**Comment by Sudip909** [comment]
Movie and drama extension please and more anime extension make the app super duper great

**Comment by ghost** [comment]
How to use the extension help me please because i installed and i only see it in the "anime extension' not in "anime source" so i can't open the "app extension" in "aniyomi"

**Reply by Sudip909** [reply]
Install the extension in one click

**Comment by Devansh7u** [comment]
Please add animesuge extention as it is the only website that I could find that give 1080p  on naruto shippuden both dub and sub.
Besides that it is a really nice app
![Uploading chrome_screenshot_1 Aug 2022 17_28_27 GMT+05_30.png…]()

**Comment by MM001-r** [comment]
Hi guys, please add to your application extensions of Russian sites, by type jut.su ; animego.org and others . thanks for making such good programs.

**Comment by AhmedAl-Otaibi** [comment]
Name: 356 show
Link: https://cinema.365ar.show/ar
Language: Arabic / العربية 

Name: fushaar
Link: https://www.fushaar.live/
Language: Arabic / العربية 

Name TopCinema
Link: https://web3.topcinema.fun/
Language: Arabic / العربية

Name Series-time
Link: https://series-time.xyz/m1/
Language: Arabic / العربية

Name cinema com 
Link: https://elcinema.com/
Language: Arabic / العربية

Name: 3isk
Link: https://jk.3ck.link/
Language: Arabic / العربية

Name: Gateanime
Link: https://s.gateanime.com
Language: Arabic / العربية

Name: Cima4u
Link: https://cima4u.film
Language: Arabic / العربية

Name: gatecima
Link: https://gatecima.com
Language: Arabic / العربية

Name: Cima4up
Link: https://cinema4up.net
Language: Arabic / العربية
